client/components/dashboard/dashboard-version-service.js

```diff
@@ -113,14 +113,21 @@ DashboardVersionService.prototype.verifyAndUpdateModel = function(dashboard) {
   // current.
   if (version == this.currentVersion) {
     if (!goog.isDef(dashboard.version)) {
+      goog.asserts.assert(!goog.isNull(version.version));
```

Comment:
Also, storing *undefined* while declaring something not nullable seems weird. I think it may be cleaner to replace *undefined* with *null* and just declare the field to be nullable.